packages/components/src/tab-panel/index.tsx

```diff
@@ -31,24 +40,27 @@ const TabButton = ( { tabId, onClick, children, selected, ...rest } ) => (
 	</Button>
 );
 
-export default function TabPanel( {
-	className,
-	children,
-	tabs,
-	initialTabName,
-	orientation = 'horizontal',
-	activeClass = 'is-active',
-	onSelect = noop,
-} ) {
+function UnconnectedTabPanel(
+	{
+		className,
+		children,
+		tabs,
+		initialTabName,
+		orientation = 'horizontal',
+		activeClass = 'is-active',
+		onSelect = noop,
```

Comment:
I wonder if we can drop the `noop` default value and just use optional chaining instead (i.e `onSelect?.()`)

Reply:
Great idea!